.vib/wordpress/goss/wordpress.yaml

```diff
@@ -0,0 +1,42 @@
+file:
```

Comment:
Looking into the postunpack logic, I would add a check for the `WEB_SERVER_DAEMON_USER/WEB_SERVER_DAEMON_GROUP`. I simple check to confirm `daemon:daemon` exists should suffice.